desktop/core/src/desktop/models.py

```diff
@@ -90,17 +92,57 @@ def hue_version():
   if HUE_VERSION is None:
     HUE_VERSION = HUE_DESKTOP_VERSION
 
+  return HUE_VERSION
+
+def hue_image_version():
+  global HUE_IMAGE_VERSION
+
+  if HUE_IMAGE_VERSION is None:
     p = get_run_root('cloudera', 'cdh_version.properties')
     if os.path.exists(p):
       build_version = _version_from_properties(open(p))
       if build_version:
-        HUE_VERSION = '%s - %s' % (HUE_VERSION, build_version)
+        HUE_IMAGE_VERSION = '%s' % (build_version)
 
-  return HUE_VERSION
+    elif os.getenv('HUE_CONF_DIR') and os.path.exists(os.getenv('HUE_CONF_DIR')):
+      cdw_path = os.getenv('HUE_CONF_DIR') + '/zhue.ini'
+      hue_img_version = _version_from_hue_conf(open(cdw_path))
+      if hue_img_version:
+        HUE_IMAGE_VERSION = '%s' % (hue_img_version)
+
+  return HUE_IMAGE_VERSION
+
+def selected_hue_vw():
+  global SELECTED_HUE_VW
+
+  if SELECTED_HUE_VW is None:
+    if os.getenv('HUE_CONF_DIR') and os.path.exists(os.getenv('HUE_CONF_DIR')):
+      cdw_path = os.getenv('HUE_CONF_DIR') + '/zhue.ini'
```

Comment:
Will this code run on non-DWX system? like CM/CDH setup?

Reply:
Yes it will run on CM/CDH setup. Tested it. Added a check to see if the cdw_path exisits. 
<img width="793" alt="Screen Shot 2023-03-10 at 9 31 37 AM" src="https://user-images.githubusercontent.com/33496652/224385962-4d940d5e-6037-4669-a3d4-8d3e5116952e.png">